Configuration/PyReleaseValidation/python/relval_2017.py

```diff
@@ -58,6 +59,7 @@
            11634.511,11634.512, # 11634.513,11634.514,
            11634.521,11634.522, # 11634.523,11634.524
            11634.24,11834.24,
+           11723.17,
```

Comment:
apparently it's not enough to add the upgrade workflow in relval_2017.py if it's not already on the matrix. @kpedro88 perhaps you know how to add a new workflow to the matrix, to make sure it's picked up correctly here? thanks in advance!

Reply:
I understodd this issue. The fact is that the workflow 11723.17 does not exist. 11723.0 yes, but it is not modified by seedingDeepCore, and so 11723.17 does not exist. For which WF the modifier is applied is defined here: https://github.com/cms-sw/cmssw/pull/32222/files#diff-8c163203cbaa64f0e0e6aa2b16346cfab3eaf84a57a008c7fa37193e8b9b2adbR310, I'm trying to add a selection based on fragment=='"QCD_Pt_1800_2400", but for now I did not manage...